src/main/java/sample/Utils/LuckyOneDiscountClient.java

```diff
@@ -0,0 +1,17 @@
+package sample.Utils;
+
+public class LuckyOneDiscountClient implements DiscountClientStrategy {
```

Comment:
Let's name the Interface Client and from the implementations let's remove the Discount part